### PR TITLE
Apply rustfmt that generally matches existing style.

### PR DESCRIPTION
**Pull request by marshallpierce**
I also deleted Cargo.lock as per [Cargo best practices](http://doc.crates.io/faq.html) (it was already in .gitignore).

This will make it easier to apply some further PRs I have in mind that are currently formatted differently.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,4 +9,6 @@
 .DS_Store
 tmp
 Cargo.lock
-1.txt
\ No newline at end of file
+1.txt
+.idea
```
Comment (marshallpierce) [inline .gitignore]: For JetBrains IDEs.

File: src/lib.rs [hunk 2]
```diff
@@ -392,7 +414,8 @@ mod tests {
     assert!(maybe_res.is_ok());
   }
 
- #[test]
+  #[test]
+  #[cfg_attr(rustfmt, rustfmt_skip)]
```
Comment (marshallpierce) [inline src/lib.rs]: This is needed so rustfmt won't mangle the long test strings.

File: rustfmt.toml [hunk 3]
```diff
@@ -0,0 +1,4 @@
+max_line_width = 120
```
Comment (marshallpierce) [inline rustfmt.toml]: This seemed to more or less approximate what was currently there. I'm open to more fiddling if you care about some of the things rustfmt has applied, though in general it defaults to the rust style guidelines.

**Comment by GildedHonour** [comment]
Removing Cargo.lock is ok, thanks, but everything else isn't.

**Comment by marshallpierce** [comment]
How open are you to other changes? I won't bother with more PRs if you're not open to other more structural changes.

**Pull request closed without merging**